src/commands/generate/__tests__/generatorTest.ts

```diff
@@ -106,6 +106,7 @@ export { integrationSecond };
 
 const irDir = path.join(__dirname, "../../../../build/ir-test-cases");
 const testCaseDir = path.join(__dirname, "resources/test-cases");
+const brandedTestCaseDir = path.join(__dirname, "resources/branded-test-cases");
```

Comment:
I was torn on duplicating the test-cases _again_ and having a whole other copy of these generated files for this `generatorTest`, but I think it's good to be clear and actually differentiate the flavored from branded. 

It does make for some noise now in this PR though, at least you can skip over anything `src/commands/generate/__tests__/resources/branded-test-cases/**` (they're also the only added files)